lib/tasks/create_customer_support_download.rake

```diff
@@ -0,0 +1,50 @@
+task create_customer_support_download: :environment do
+  ### This is a utility that serves as a kind of "customer support dropbox".
+  ### The task will create a BulkDownload record that the user can use to download a file you wish to transfer to them.
+  ### The task will print instructions on the S3 location where you should put the file you have created for the user.
+  ### Usage: DESTINATION_USER_ID=<int> rake create_customer_support_download
+
+  user_id = ENV['DESTINATION_USER_ID'].to_i
+
+  user = User.find(user_id)
+  is_user_correct = await_confirmation("Is #{user.name} (#{user.email}) the user you wish to send your file to?",
+                                       "Please verify your DESTINATION_USER_ID and start over")
+  exit 2 unless is_user_correct
+
+  puts "Please enter a user-facing description of the file you wish to transfer:"
+  description = ActionController::Base.helpers.strip_tags(STDIN.gets.strip)
+
+  bulk_download = BulkDownload.create(download_type: "customer_support_request",
+                                      status: BulkDownload::STATUS_WAITING,
+                                      description: description,
+                                      user_id: user.id)
+  puts "BulkDownload record of type customer_support_request has been created."
+  s3_uri = "s3://#{ENV['SAMPLES_BUCKET_NAME']}/#{bulk_download.download_output_key}"
```

Comment:
Because of how `download_output_key` is currently implemented, the admin operator will need to name their file `Customer Support Request.tar.gz`, regardless of what the file is. This is because BulkDownloads originally assumed that all files would be like this.

Since this is an admin-only feature, maybe this is okay, but I have ideas for how to make this better. Let me know if you want to discuss.

Reply:
Very true. I'd love to hear your ideas on this! I see it as becoming especially problematic when the operator seeks to transfer a file that's not actually a tar.gz -- in this case, the extension could be very confusing for the user